authors/tests/base_test.py

```diff
@@ -8,6 +8,7 @@
 # local imports
 from authors.apps.authentication.activate import account_activation_token
 from authors.apps.authentication.models import User
+import os
```

Comment:
I read somewhere such an import should at the top of the file, ie os imports come first, the third party then local imports
And you have put it under the comment 'local imports', kindly look in to that